doc/0_quick_start.md

```diff
@@ -17,6 +17,14 @@ Configure a new Scala project with the following dependencies:
  - Apache Cassandra thrift and clientutil libraries matching the version of Cassandra  
  - DataStax Cassandra driver for your Cassandra version 
  
+For example, if you're using `sbt` and run Cassandra 2.2.5 and Spark 1.6.1, your build.sbt should include something like this:
+
+    libraryDependencies += "com.datastax.spark" %% "spark-cassandra-connector" % "1.6.0"
+    libraryDependencies += "org.apache.spark" %% "spark-core" % "1.6.1"
+    libraryDependencies += "com.datastax.cassandra" % "cassandra-driver-core" % "3.0.0"
```

Comment:
All the dependencies below here

cassandra-driver-core, cassandra-thrift, and cassandra-clientutil 

Are not necessary also I would recommend we add an example pulling form spark packages instead of this

Something like

sbt
```

If you use the sbt-spark-package plugin, in your sbt build file, add:

spDependencies += "datastax/spark-cassandra-connector:1.5.1-s_2.11"
Otherwise,

resolvers += "Spark Packages Repo" at "http://dl.bintray.com/spark-packages/maven"

libraryDependencies += "datastax" % "spark-cassandra-connector" % "1.5.1-s_2.11"
```

Which is auto generated on the spark packages website